tests/suites/test_suite_pkparse.function

```diff
@@ -101,6 +101,10 @@ void pk_parse_keyfile_ec(char *key_file, char *password, int result)
     mbedtls_pk_context ctx;
     int res;
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    PSA_INIT();
+#endif
```

Comment:
Minor: there's a shortcut for that: `USE_PSA_INIT();` (same for DONE of course).

Note: it's usually a bad compatibility smell to have to add this to a test. (One of our big compatibility challenges until 4.0 is we can't start requiring calls to `psa_crypto_init()` in places and configs that didn't require it before.) However in that case, we've been documenting that when you enable `USE_PSA` you need to call `psa_crypto_init()` before using any PK function, so the requirement was already documented. Things just happened to work without it, but our compatibility promise only covers documented behaviour, so it doesn't matter that it just happened to work.

Which is a long way of saying, we're fine here. But we should be careful in general.